app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -31,6 +34,20 @@ def new_credit_card_id
     register_error(:credit_card_key, :credit_card_invalid)
   end
 
+  def charge(subscription)
+    RecurringContribution::Subscriptions::Create.process(subscription)
+  end
+
+  def charge_scheduled_for_today?
+    (subscription.charging_day == DateTime.current.day) && with_save
+  end
+
+  def subscription_status
+    return :pending_payment if charge_scheduled_for_today?
```

Comment:
Faz sim, @maychell. Pq a assinatura só é criada no pagarme se a assinatura da juntos for salva(isso é definido pelo `with_save`)